languages/cpp/exercises/concept/numbers/.docs/introduction.md

```diff
@@ -0,0 +1,73 @@
+There are two different types of numbers in C++: integers and floats.
+
+## Integer numbers
+
+The keyword to define an integer type is `int`. The keyword `int` can be omitted if one of the length modifiers is present. The actual width in bits depends on the data model (computer architecture).
+
+This type can be extended with sign and length modifiers. Modifiers can be mixed in any order. Only one of each group can be present in type name.
+
+Signedness:
+
+- `signed` - target type will have signed representation (this is the default if omitted)
+- `unsigned` - target type will have unsigned representation
+
+Size:
+
+- `short` - target type will be optimized for space and will have width of at least 16 bits.
+- `long` - target type will have width of at least 32 bits.
+- `long long` - target type will have width of at least 64 bits.
+
+**Note**: integer arithmetic is defined differently for the signed and unsigned integer types. See arithmetic operators, in particular integer overflows.
+
+**Note**: integers are used in boolean expression context, `0` is `false`, everything else is `true`.
+
+### Arithmetic operators
+
+Arithmetic is done using the standard [arithmetic operators][arithmetic-operators] (`+`, `-`, `*`, etc.). Numbers can be compared using the standard [comparison and equality operators][comparison-operators] (`<`, `>=`, etc.) and the equality-operator (`==`) and inequality operator (`!=`).
+
+### Fixed width integer types
+
+By including `<cstdint>`, programmer gains access to [integer types with fixed width][fixed-width-integer-types]. This provides data-model independent way of representing integer numbers and ensures consistent results of arithmetic operations across all platforms.
```

Comment:
Missing articles:
"the programmer" or "a programmer"
"provides a data-model"
But (disclaimer:) I'm not a native English speaker.